R/flex_dashboard.R

```diff
@@ -55,7 +55,7 @@
 #'  each dashboard component in a navigable storyboard frame. When a storyboard
 #'  layout is used the \code{orientation} and \code{vertical_layout} arguments
 #'  are ignored. When creating a dashbaord with multiple pages you should apply
-#'  the `{.storyboard}` attribute to individual pages rather than using the
+#'  the `\{.storyboard\}` attribute to individual pages rather than using the
```

Comment:
I guess flexdashboard isn't using roxygen2 markdown, which is the real source of this issue.

```suggestion
#'  the \code{\{.storyboard\}} attribute to individual pages rather than using the
```